### DISCUSSION TITLE
v9.0.0-alpha.0

### DISCUSSION THREAD

**Discussion by thymikee** [react-native-community/cli#1610] [Announcements]
## What's Changed
* feat: autolinking for the new architecture on Android by @thymikee in https://github.com/react-native-community/cli/pull/1603

**Full Changelog**: https://github.com/react-native-community/cli/compare/v8.0.0...v9.0.0-alpha.0

<hr /><em>This discussion was created from the release <a href='https://github.com/react-native-community/cli/releases/tag/v9.0.0-alpha.0'>v9.0.0-alpha.0</a>.</em>

**Comment by matinzd** [comment]
Let's pick https://github.com/react-native-community/cli/commit/2a314d3fb41428f5f1d554a3a1054baa46a5344c

**Reply by thymikee** [reply]
It's already out in v8 and v9 :)
